### ISSUE TITLE
Live chat module not translated

### ISSUE THREAD

**Issue by ecomturbo** [freescout-helpdesk/freescout#1379]
We've paid for the live chat module but found out it can't be translated.
This should be mentioned on the sales page please since you don't accept refunds or reply to support tickets.
Just saying this to show a little gratitude to buyers who invest into freescout.

**Comment by freescout-helpdesk** [comment]
Modules can be tested on https://demo.freescout.net

**Comment by ecomturbo** [comment]
Can you confirm that we can't translate the text within the live chat module? We would like to translate "Chat with us", "Please update your info", "Type a message here...". This would help us. Thanks.

**Comment by freescout-helpdesk** [comment]
See "How can I translate modules?"
https://github.com/freescout-helpdesk/freescout/wiki/FreeScout-Modules#how-can-i-translate-modules

**Comment by ecomturbo** [comment]
We've already done that but no where can we find the live chat text. Can you tell us in which group is the live chat module located?
We've check everthing under "app" and each and every module. Thank you.

**Comment by freescout-helpdesk** [comment]
Activate the module and:

![2021-08-21_08-08-00](https://user-images.githubusercontent.com/40499291/130311217-850865e7-8f34-4c2f-9b4f-c844c983b6b2.png)

**Comment by ecomturbo** [comment]
We don't see chat in our list despite the module being activated.

**Comment by freescout-helpdesk** [comment]
You can send translations right here, we'll add them to the module.

:%a_begin%End-User Portal Module:%a_end% is required.
After making updates in the settings you need to update the code on your website.
Attachment
Bottom-left corner of the page
Bottom-right corner of the page
Chat Operating Hours
Chat operating hours allow to display 'Contact Us' form instead of a chat when your support agents are not working.
Chat operating hours are based on your :%a_begin%company timezone:%a_end%.
Chat with us
Contact us
Edit my info
Edit Your Info
Friday
Get the Code
If you are embedding the contact form widget on HTTPS website, your FreeScout must also use HTTPS.
Main Color
Monday
Not delivered
Please update your info
Position
Saturday
Submit another message
Sunday
Thursday
Tuesday
Type a message here
Visitor
Wednesday
Widget
Your message has been sent!

**Comment by ecomturbo** [comment]
Hello, Thank you! Please find enclosed our translation into French.

:%a_begin%End-User Portal Module:%a_end% is required.
:%a_begin%Module du portail utilisateur final :%a_end% est requis.
After making updates in the settings you need to update the code on your
website.
Après avoir effectué des mises à jour dans les paramètres, vous devez
mettre à jour le code sur votre site Web.
Attachment Pièce jointe
Bottom-left corner of the page
Coin inférieur gauche de la page
Bottom-right corner of the page
Coin inférieur droit de la page
Chat Operating Hours
Heures d'ouverture du chat
Chat operating hours allow to display 'Contact Us' form instead of a chat
when your support agents are not working.
Les heures d'ouverture du chat permettent d'afficher le formulaire
"Contactez-nous" au lieu d'un chat lorsque vos agents de support ne
travaillent pas.
Chat operating hours are based on your :%a_begin%company timezone:%a_end%.
Les heures d'ouverture du chat sont basées sur votre fuseau
horaire :%a_begin%company :%a_end%.
Chat with us
Discutez avec nous
Contact us Contactez-nous
Edit my info
Modifier mes informations
Edit Your Info
Modifier vos informations
Friday Vendredi
Get the Code Obtenez le code
If you are embedding the contact form widget on HTTPS website, your
FreeScout must also use HTTPS.
Si vous intégrez le widget de formulaire de contact sur le site Web HTTPS,
votre FreeScout doit également utiliser HTTPS.
Main Color
Couleur principale
Monday Lundi
Not delivered Non livrés
Please update your info
Veuillez mettre à jour vos informations
Position Position
Saturday Samedi
Submit another message
Envoyer un autre message
Sunday Dimanche
Thursday Jeudi
Tuesday Mardi
Type a message here
Entrez votre message ici
Visitor Visiteur
Wednesday Mercredi
Widget Widget
Your message has been sent!
Votre message a été envoyé !

**Comment by freescout-helpdesk** [comment]
French translation added in the Chat Module v1.0.5

**Comment by ecomturbo** [comment]
Thank you very much! Appreciate your help.

**Comment by Michal888888** [comment]
Hello, I have added Czech translation, but it Is still showing English - can you please help me with this?
<img width="1030" alt="image" src="https://user-images.githubusercontent.com/110168681/182129524-77d18873-e77f-493b-b9ff-185bc50e3e43.png">

**Comment by Michal888888** [comment]
adding the translation list - in case you will upload Tham manually:


Key | cs
-- | --
:%a_begin%End-User Portal Module:%a_end% is required. | :%a_begin%Modul pro zákazníky:%a_end% je požadován.
After making updates in the settings you need to update the code on your website. | Po úpravě nastavení je potřeba aktualizovat kód na Vaší stránce.
Attachment | Příloha
Bottom-left corner of the page | Levý spodní roh stránky
Bottom-right corner of the page | Pravý spodní roh stránky
Chat Operating Hours | Pracovní doba chatu
Chat operating hours allow to display 'Contact Us' form instead of a chat when your support agents are not working. | Pracovní doba umožní zobrazení 'kontaktního formuláře' místo chatu v mimo pracovní doby Vašich agentů.
Chat operating hours are based on your :%a_begin%company timezone:%a_end%. | Pracovní doba chatu je závislá na :%a_begin%Vaší časové zóně:%a_end%.
Chat with us | Napište nám
Contact us | Kontaktujte nás
Day off | Volno
Edit my info | Upravit mé kontakty
Edit Your Info | Upravte své kontakty
Friday | Pátek
Get the Code | Zde je Váš kód
If you are embedding the contact form widget on HTTPS website, your FreeScout must also use HTTPS. | Pokud vkládáte kontaktní formulář na HTTPS stránku, Váš FreeScout musí také používat HTTPS.
Main Color | Hlavní barva
Monday | Pondělí
Not delivered | Nedoručeno
Open in New Window | Otevřít v novém okně
Please update your info | Prosím aktualizujte své údaje
Position | Pozice
Saturday | Sobota
Submit another message | Odeslat další zprávu
Sunday | Neděle
Thursday | Čtvrtek
Tuesday | Úterý
Type a message here | Zde napište zprávu
Visitor | Návštěvník
Visitor can be optionally set to pass visitor info to the widget on the website. | Zákazník může být na přání přesunut na widget stránky.
Wednesday | Středa
Widget | Widget
Your message has been sent! | Vaše zpráva byla odeslána!

**Comment by Michal888888** [comment]
thank you
Michal

**Comment by Michal888888** [comment]
2nd question: 
Is there any code that would ensure that each customer has to enter their email before starting a chat, so that we can contact them later? 
Thank you
